containers/shop/product-card.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div class="card product with-image">
+    <Button
+      v-if="closable"
+      @click="closeModal"
+      normal
+      round
+      class="close"
+      img="/images/x.svg"
+    />
+    <Carousel :images="activeVariety.images" />
```

Comment:
When a carousel is on the 2nd picture, then a variety (color) with only one picture is chose, the following happens:
![image](https://user-images.githubusercontent.com/11016151/62326358-30f23680-b4ae-11e9-8d95-4e85771a1408.png)
This is because the "current" variable inside the carousel stays on 1 and doesn't reset to 0
Suggestion: add a watcher inside the carousel component to reset to 0 if the `images` prop changes
Suggestion 2: expose the "current" variable as a prop (maybe even `v-model`)

Reply:
Done